### ISSUE TITLE
Not working at spec files

### ISSUE THREAD

**Issue by mestihudson** [SanderRonde/prettier-plugin-sort-imports#3]
I get make it work in my `src/**/*.ts` files but not in my `tests/**/*.spec.ts`.

Before
```typescript
import { Test, TestingModule } from '@nestjs/testing';
import * as request from 'supertest';
import { INestApplication, HttpStatus, ValidationPipe } from '@nestjs/common';
import { HttpAdapterHost, NestFactory } from '@nestjs/core';
import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
import * as cookieParser from 'cookie-parser';
import * as csurf from 'csurf';
import * as fs from 'fs';
import helmet from 'helmet';
import { AppModule } from '@/app.module';
import { AllExceptionsFilter } from '@/infra/http/filters/all-exceptions.filter';
import { ExceptionInterceptor } from '@/infra/http/interceptors/exception.interceptor';
import { NewrelicInterceptor } from '@/infra/newrelic/newrelic.interceptor';

describe('smoke', () => {
    it('should be successful', async () => {
        expect(true).toBeTruthy();
    });
});
```

After
```typescript
import { INestApplication, HttpStatus, ValidationPipe } from '@nestjs/common';
import { HttpAdapterHost, NestFactory } from '@nestjs/core';
import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
import * as cookieParser from 'cookie-parser';
import * as csurf from 'csurf';
import * as fs from 'fs';
import helmet from 'helmet';

import { AppModule } from '@/app.module';
import { AllExceptionsFilter } from '@/infra/http/filters/all-exceptions.filter';
import { ExceptionInterceptor } from '@/infra/http/interceptors/exception.interceptor';
import { NewrelicInterceptor } from '@/infra/newrelic/newrelic.interceptor';
import { Test, TestingModule } from '@nestjs/testing';
import * as request from 'supertest';

describe('smoke', () => {
    it('should be successful', async () => {
        expect(true).toBeTruthy();
    });
});
```

Expected
```typescript
import { INestApplication, HttpStatus, ValidationPipe } from '@nestjs/common';
import { HttpAdapterHost, NestFactory } from '@nestjs/core';
import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
import { Test, TestingModule } from '@nestjs/testing';
import * as cookieParser from 'cookie-parser';
import * as csurf from 'csurf';
import * as fs from 'fs';
import helmet from 'helmet';
import * as request from 'supertest';

import { AppModule } from '@/app.module';
import { AllExceptionsFilter } from '@/infra/http/filters/all-exceptions.filter';
import { ExceptionInterceptor } from '@/infra/http/interceptors/exception.interceptor';
import { NewrelicInterceptor } from '@/infra/newrelic/newrelic.interceptor';

describe('smoke', () => {
    it('should be successful', async () => {
        expect(true).toBeTruthy();
    });
});
```

package.json
```json
{
    ...
    "dependencies": {
        ...
        "@nestjs/common": "^8.0.0", 
        "@nestjs/core": "^8.0.0",
        "@nestjs/swagger": "^5.2.1",
        "cookie-parser": "^1.4.6",
        "csurf": "^1.11.0",
        "helmet": "^5.0.2",
        ...
    },
    "devDependencies": {
        ...
        "@nestjs/testing": "^8.0.0",
        "supertest": "^6.1.3",
        ...
    }
}
```

.prettierrc
```json
{
  "singleQuote": true,
  "trailingComma": "all",
  "sortingMethod": "alphabetical",
  "stripNewlines": true,
  "importTypeOrder": ["NPMPackages", "localImports"],
  "newlineBetweenTypes": true,
}
```

Environment
```text
node: 16.14.2
npm: 8.8.0
```

**Comment by mestihudson** [comment]
I believe that problem lives here (https://github.com/SanderRonde/prettier-plugin-sort-imports/blob/e4649c99f132b8f78acf08325bb95a021ea636d4/app/sorters/import-type.ts#L31-L35).
This range of code only take in account `packageJSON.dependencies`, when, in really, it should consider, also, `packageJSON.devDependencies`.
I tried to mount a dev env to make a pull request but ava test framework give me a spanking.

**Comment by SanderRonde** [comment]
Fixed by #4

**Comment by SanderRonde** [comment]
Fix is live now under version `1.6.1`
